File: swagger/models/appointment_option.py
from .base_model import BaseModel


class AppointmentOption(BaseModel):
	"""
	An appointment option name/value pair

	Attributes:

		display_name
		The name displayed for this appointment option.

		name
		The name given to this option.

		value
		The value of the option.

		type
		The data type of the option value.
	"""

	input_map = {
		'DisplayName': 'display_name',
		'Name': 'name',
		'Value': 'value',
		'Type': 'type',
		}

	output_map = {
		'display_name': 'DisplayName',
		'name': 'Name',
		'value': 'Value',
		'type': 'Type',
		}

File: swagger/Models/AppointmentOption.php
<?php

namespace App\MindBody\Models;

/**
 * An appointment option name/value pair
 */
class AppointmentOption extends BaseModel
{
	/**
	 * The name displayed for this appointment option.
	 * 
	 * @var string
	 */
	public $displayName;
	/**
	 * The name given to this option.
	 * 
	 * @var string
	 */
	public $name;
	/**
	 * The value of the option.
	 * 
	 * @var string
	 */
	public $value;
	/**
	 * The data type of the option value.
	 * 
	 * @var string
	 */
	public $type;

	protected function getInputMap()
	{
		return [
			'DisplayName' => 'displayName',
			'Name' => 'name',
			'Value' => 'value',
			'Type' => 'type',
			];
	}

	protected function getOutputMap()
	{
		return [
			'displayName' => 'DisplayName',
			'name' => 'Name',
			'value' => 'Value',
			'type' => 'Type',
			];
	}
}
